components/server/src/utils.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{io, sync::Arc};
+
+use encryption::{BackupEncryptionManager, DataKeyManager, MultiMasterKeyBackend};
+use encryption_export::create_async_backend;
+use tikv::config::TikvConfig;
+
+pub async fn build_backup_encryption_manager(
+    config: TikvConfig,
```

Comment:
right, I think it will have the cyclic dep issue, the src/server imports the encryption package, so encryption cannot use src/config anymore. Ideally should be in a separate package at the lowest level in the dep chain.

I guess we can pass in those parameters in the config separately to bypass this issue, but that's not very elegant and in future if more fields added to the config we need to change the method signature.

I'm open to either option, let me know your thoughts!